### PR TITLE
Fixing issues with timeouts

### PR DESCRIPTION
**Pull request by jakebrown58**
Timeouts were falsely timing out on the uninstallerhelper.   still need
to investigate, but this works around the issue for now.
Also fixes bad defaults for powershell and exeFile runner.

### CODE REVIEW

File: src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs [hunk 1]
```diff
@@ -10,41 +10,38 @@ namespace UninstallerHelper.App
 {
     public class UninstallCommandGenerator
     {
-        internal static string CreateUninstallString(Logger l, RegistryKey app, string matchBy, string[] exclusions)
+
+        internal static string CreateUninstallString(Logger l, RegistryFacade app, string matchBy, string[] exclusions)
```
Comment (jsnmc) [inline src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs]: Very minor issue but I might change the variable name to logger to be consistent with the rest of the code ( and I keep reading this as 1.

File: src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs [hunk 2]
```diff
@@ -10,41 +10,38 @@ namespace UninstallerHelper.App
 {
     public class UninstallCommandGenerator
```
Comment (jsnmc) [inline src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs]: Minor request - Might add a brief comment on the class level for UninstallCommandGenerator and RegistryFacade.

File: src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs [hunk 3]
```diff
@@ -159,4 +159,44 @@ public static List<RegistryKey> GetAllRingtailKeys()
         }
 
     }
+
+    public class RegistryFacade
```
Comment (jsnmc) [inline src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs]: Minor request - Might add a brief comment on the class level for UninstallCommandGenerator and RegistryFacade.

**Comment by jsnmc** [comment]
Looks good.

File: src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs (hunk 3 above)
Comment (jakebrown58) [inline src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs]: good thoughts - i might scrap a bunch of this - as it wasn't the root issue.

File: src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs (hunk 1 above)
Comment (jakebrown58) [inline src/AutoDeploy/UninstallerHelper/App/UninstallCommandGenerator.cs]: ha - good point!

**Pull request closed without merging**